index.js

```diff
@@ -20,7 +20,9 @@ CssoFilter.prototype.extensions = ['css'];
 CssoFilter.prototype.targetExtension = 'css';
 
 CssoFilter.prototype.processString = function (str) {
-	return csso.justDoIt(str, this.options.restructure);
+	return csso.minify(str, {
+		restructuring: this.options.restructure
```

Comment:
Just pass `this.options` directly to `css.minify` so it's forward compatible for additional options.